packages/lit-query/src/createQueriesController.ts

```
@@ -0,0 +1,486 @@
+import {
+  QueriesObserver,
+  type DefaultError,
```

ESLint

[error] Member 'DefaultError' of the import declaration should be sorted alphabetically.

(sort-imports)